runtime/rasdump/javadump.cpp

```diff
@@ -5165,6 +5179,64 @@ JavaCoreDumpWriter::writeCPUinfo(void)
 	return;
 }
 
+#if defined(LINUX)
+void
+JavaCoreDumpWriter::writeCgroupMetrics(void)
+{
+	OMRPORT_ACCESS_FROM_J9PORT(_PortLibrary);
+	BOOLEAN isCgroupSystemAvailable = omrsysinfo_cgroup_is_system_available();
+	if (isCgroupSystemAvailable) {
+		const OMRCgroupEntry *entryHead = omrsysinfo_get_cgroup_subsystem_list();
+		OMRCgroupEntry *cgEntry = (OMRCgroupEntry *)entryHead;
+		int32_t rc = 0;
+		if (NULL != cgEntry) {
+			_OutputStream.writeCharacters("NULL \n");
+			_OutputStream.writeCharacters("1CICGRPINFO    Cgroup Information \n");
+			_OutputStream.writeCharacters("NULL           ------------------------------------------------------------------------\n");
+			do {
+				_OutputStream.writeCharacters("2CICGRPINFO    subsystem : ");
+				_OutputStream.writeCharacters(cgEntry->subsystem);
+				_OutputStream.writeCharacters("\n");
+				_OutputStream.writeCharacters("2CICGRPINFO    cgroup name : ");
+				_OutputStream.writeCharacters(cgEntry->cgroup);
+				_OutputStream.writeCharacters("\n");
+                                OMRCgroupMetricIteratorState cgroupState = {0};
+				rc = omrsysinfo_cgroup_subsystem_iterator_init(cgEntry->flag, &cgroupState);
+				if (0 == rc) {
+					if (0 != cgroupState.numElements) {
+						OMRCgroupMetricElement metricElement = {0};
+						while (0 != omrsysinfo_cgroup_subsystem_iterator_hasNext(&cgroupState)) {
+							const char *metricKey = NULL;
+							rc = omrsysinfo_cgroup_subsystem_iterator_metricKey(&cgroupState, &metricKey);
+							if (0 == rc) {
+								rc = omrsysinfo_cgroup_subsystem_iterator_next(&cgroupState, &metricElement);
+								if (rc == 0) {
+									_OutputStream.writeCharacters("2CICGRPINFO		");
```

Comment:
This string literal should not use tabs.
Also, I think this logical level should use a higher leading number: `3CICGRPINFO` (here and the alternate below).

Reply:
Made changes to the label. and replaced tabs with spaces.